go/api-frontend/aaa/multi_token_backend.go

```diff
@@ -0,0 +1,63 @@
+package aaa
+
+import (
+	"time"
+)
+
+type MultiTokenBackend struct {
+	backends []TokenBackend
+}
+
+func NewMultiTokenBackend(backends ...TokenBackend) TokenBackend {
+	return &MultiTokenBackend{
+		backends: append([]TokenBackend{}, backends...),
+	}
+}
+
+func (tb *MultiTokenBackend) TokenInfoForToken(token string) (*TokenInfo, time.Time) {
+	for _, b := range tb.backends {
+		token, t := b.TokenInfoForToken(token)
+		if token != nil {
+			return token, t
+		}
+	}
+	return nil, time.Unix(0, 0)
+}
+
+func (tb *MultiTokenBackend) StoreTokenInfo(token string, ti *TokenInfo) error {
+	var errs []error
+	for _, b := range tb.backends {
+		err := b.StoreTokenInfo(token, ti)
+		if err != nil {
+			errs = append(errs, err)
+		}
+	}
+	return nil
```

Comment:
It should be returning an error when errs is not empty

Reply:
Or if we can't return an error because that would prevent read-only from working, we could:
- Return an error when `len(errs) == len(backends)`
- Log warnings regardless so that we know backend xyz produced an error

Reply:
Ok done